LICENSE.txt

```diff
@@ -1,29 +1,62 @@
-The src/c_gsw subdirectory contents are derived from the GSW-C
-project, and are subject to src/c_gsw/LICENSE.
-
-Sections of documentation text, as well as the function names and the
-names of their inputs and outputs, and the algorithms, are ultimately
-derived from the GSW-Matlab implementation and other work of the
-TEOS-10 developers, and may be subject to additional restrictions.
-See http://teos-10.org/ for more information.
-
-Copyright Notice and Statement for the remainder of the gsw project,
-which is this specific Python implementation:
-
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in
-all copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-THE SOFTWARE.
+Licence for the use of the Gibbs SeaWater (GSW) Oceanographic Toolbox
+
+Copyright (c) 2011, SCOR/IAPSO WG127 (Scientific Committee on Oceanic Research/
+International Association for the Physical Sciences of the Oceans, Working Group 127).
+
```

Comment:
I was thinking the 2017-2020 copyright line would go here, immediately following the 2 lines above.

Reply:
Did some digging, seems standard practice is to simply add your name as a name to the copyright section and include all the original BSD license text. So basically, just use the GSW-C license file with only slight modifications to the headers/copyright file.

Reply:
Are you suggesting that the modification as you have it now is correct, or are you saying it should be changed?  (It looks OK to me.)

Reply:
I think I have it as "correct" as my non lawyer eyes can make it. Should have been clear that I updated the PR to reflect this change.